frontend/components/SummaryTable.js

```diff
@@ -0,0 +1,76 @@
+const SummaryTable = (props) => {
+    const yearData = props.data.data['2018'];
+    const percentAssisted = `${(yearData["% of Households below 200% FPL Assisted"] * 100).toFixed(2)}%`;
+    const selectedYear = '2018';
+
+    return (
+        <div className="summary-table">
+            <h3>{selectedYear} Overview</h3>
+            <table>
+            <thead>
+                <tr>
+                    <td></td>
+                    <td></td>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>Low Income Households</td>
+                    <td className="data">{ withComma(yearData["Households below 200% FPL"])}</td>
+                </tr>
+                <tr>
+                    <td>Total Households Assisted</td>
+                    <td className="data">{ withComma(yearData["Total Households Assisted"])} </td>
+                </tr>
+                <tr>
+                    <td>% of LI Households Assisted</td>
+                    <td className="data">{percentAssisted}</td>
+                </tr>
+            </tbody>
+            </table>
+            <style jsx>{`
```

Comment:
The table looks good!  If it makes sense to use in the main table as well I think we should move this up to its own component and import it.